FEATURES.md

```diff
@@ -0,0 +1,113 @@
+- [Go to Definition](#go-to-definition)
+    - [Go to Active Record DSL Definitions](#go-to-active-record-dsl-definitions)
+    - [Go to Active Record Associations](#go-to-active-record-associations)
+    - [Go to Route Helper Definitions](#go-to-route-helper-definitions)
+- [Ruby File Operations](#ruby-file-operations)
+    - [Commands](#commands)
+        - [Rails Generate](#rails-generate)
+        - [Rails Destroy](#rails-destroy)
+    - [Icon](#icon)
+- [Run and Debug](#run-and-debug)
+    - [Run Tests With Test Explorer](#run-tests-with-test-explorer)
+    - [Run Tests In The Terminal](#run-tests-in-the-terminal)
+    - [Debug Tests With VS Code](#debug-tests-with-vs-code)
+
+## Document Symbol
+
+Document Symbol is a way to represent the structure of a document. They are used to provide a quick overview of the
+document and to allow for quick navigation.
+
+Ruby LSP already provides document symbols for Ruby files, such as classes, modules, methods, etc. But the Rails addon
+provides additional document symbols for Rails specific features.
+
+In VS Code, you can open the document symbols view by pressing `Ctrl + Shift + O`.
+
+### Active Record DSLs and Associations
+
+Navigates between Active Record DSLs and Associations using the `Document Symbol` feature.
+
+![Document Symbol for Active Record DSLs and Associations](./ruby-lsp-rails-document-symbol-ar-model.gif)
+
+### Active Support Test Cases
+
+Navigates between Active Support test cases using the `Document Symbol` feature.
+
+![Document Symbol for tests](./ruby-lsp-rails-test-document-symbol.gif)
+
+## Go to Controller Action Route
+
+Navigates to the route definition of a controller action using the `Code Lens` feature.
+
+![Go to Controller Action Route](./ruby-lsp-rails-controller-action-to-route.gif)
+
+## Go to Controller Action View
+
+Navigates to the view file(s) of a controller action using the `Code Lens` feature.
+
+![Go to Controller Action View](./ruby-lsp-rails-controller-action-to-view.gif)
+
+## Go to Definition
+
+Go to definition is a feature that allows you to navigate to the definition of a symbol.
+
+In VS Code, you can trigger go to definition in 3 different ways:
+
+- Select `Go to Definition` from the context menu
+- `F12` on a symbol
+- `Cmd + Click` on a symbol
+
+In the following demos, we will use the `Cmd + Click` method to trigger go to definition.
+
+### Go to Active Record DSL Definitions
+
+![Go to Active Record DSL Definitions](./ruby-lsp-rails-go-to-ar-dsl-definitions.gif)
+
+### Go to Active Record Associations
+
+![Go to Active Record Associations](./ruby-lsp-rails-go-to-ar-associations.gif)
+
+### Go to Route Helper Definitions
+
+![Go to Route Helper Definitions](./ruby-lsp-rails-go-to-route-definitions.gif)
+
+## Ruby File Operations
+
+### Commands
+
+#### Rails Generate
+
+![Rails Generate](./ruby-lsp-rails-generate-command.gif)
+
+#### Rails Destroy
+
+![Rails Destroy](./ruby-lsp-rails-destroy-command.gif)
+
+### Icon
```

Comment:
This section being called icon is a bit odd. Would it make sense to move the content directly under `Ruby File Operations`?